src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,131 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: $card-gap-vertical $card-gap-horizontal;
+  padding: $catalog-padding-vertical $catalog-padding-horizontal;
+
+  @media (min-width: 488px) {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+}
+
+.card {
+  background: $color-bg-white;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  position: relative;
+  border-radius: 8px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  transition:
+    transform 0.3s ease,
+    box-shadow 0.3s ease;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    color: $color-text-dark;
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin: 0;
+    transition: color 0.3s ease;
+
+    &:hover {
+      transform: scale(1.2);
+      z-index: 10;
+
+      .card__title {
+        color: #34568b;
+      }
```

Comment:
The nesting of '.card__title' inside its own ':hover' pseudo-class is incorrect. The '.card__title' should not be nested within ':hover' as it is already being targeted. Consider removing the nested '.card__title' and directly applying the color change within the ':hover' block.